simpa/core/device_digital_twins/detection_geometries/ithera_invision_array.py

```diff
@@ -0,0 +1,353 @@
+[-0.03833734, 0.01305751, 0.],
+[-0.03857119, 0.01234963, 0.],
+[-0.03879197, 0.01163757, 0.],
+[-0.03899961, 0.01092157, 0.],
+[-0.03919403, 0.01020186, 0.],
+[-0.03937517, 0.0094787, 0.],
+[-0.03954297, 0.00875233, 0.],
+[-0.03969738, 0.00802299, 0.],
+[-0.03983833, 0.00729093, 0.],
+[-0.03996578, 0.0065564, 0.],
+[-0.04007969, 0.00581965, 0.],
+[-0.04018002, 0.00508093, 0.],
+[-0.04026674, 0.00434048, 0.],
+[-0.04033981, 0.00359857, 0.],
+[-0.04039921, 0.00285544, 0.],
+[-0.04044493, 0.00211133, 0.],
+[-0.04047694, 0.00136652, 0.],
+[-0.04049524, 0.00062124, 0.],
+[-0.04049981, -0.00012425, 0.],
+[-0.04049066, -0.0008697, 0.],
+[-0.04046779, -0.00161485, 0.],
+[-0.04043121, -0.00235946, 0.],
+[-0.04038093, -0.00310327, 0.],
+[-0.04031697, -0.00384602, 0.],
+[-0.04023935, -0.00458747, 0.],
+[-0.04014809, -0.00532737, 0.],
+[-0.04004323, -0.00606546, 0.],
+[-0.0399248, -0.0068015, 0.],
+[-0.03979284, -0.00753523, 0.],
+[-0.0396474, -0.00826641, 0.],
+[-0.03948853, -0.00899479, 0.],
+[-0.03931627, -0.00972012, 0.],
+[-0.0391307, -0.01044216, 0.],
+[-0.03893186, -0.01116066, 0.],
+[-0.03871983, -0.01187538, 0.],
+[-0.03849469, -0.01258607, 0.],
+[-0.0382565, -0.0132925, 0.],
+[-0.03800534, -0.01399442, 0.],
+[-0.03774131, -0.01469161, 0.],
+[-0.03746449, -0.01538381, 0.],
+[-0.03717498, -0.0160708, 0.],
+[-0.03687287, -0.01675235, 0.],
+[-0.03655827, -0.01742822, 0.],
+[-0.03623128, -0.01809819, 0.],
+[-0.03589201, -0.01876202, 0.],
+[-0.03554058, -0.0194195, 0.],
+[-0.03517711, -0.02007039, 0.],
+[-0.03480172, -0.02071449, 0.],
+[-0.03441454, -0.02135156, 0.],
+[-0.0340157, -0.02198141, 0.],
+[-0.03360533, -0.0226038, 0.],
+[-0.03318357, -0.02321854, 0.],
+[-0.03275058, -0.0238254, 0.],
+[-0.03230648, -0.0244242, 0.],
+[-0.03185144, -0.02501472, 0.],
+[-0.0313856, -0.02559676, 0.],
+[-0.03090913, -0.02617013, 0.],
+[-0.03042219, -0.02673463, 0.],
+[-0.02992494, -0.02729007, 0.],
+[-0.02941755, -0.02783627, 0.],
+[-0.02890019, -0.02837304, 0.]]) * 1000)[:, [0, 2, 1]]
+
+        detector_positions = self.get_detector_element_positions_base_mm()
+        min_x_coordinate = np.min(detector_positions[:, 0])
+        max_x_coordinate = np.max(detector_positions[:, 0])
+        self.probe_width_mm = max_x_coordinate - min_x_coordinate
+
+        min_z_coordinate = np.min(detector_positions[:, 2])
+        max_z_coordinate = np.max(detector_positions[:, 2])
+        self.probe_height_mm = max_z_coordinate - min_z_coordinate
+
+        if field_of_view_extent_mm is None:
+            self.field_of_view_extent_mm = np.asarray([-self.probe_width_mm/2,
+                                                       self.probe_width_mm/2,
+                                                       0, 0, 0, 100])
+        else:
+            self.field_of_view_extent_mm = field_of_view_extent_mm
+
+    def check_settings_prerequisites(self, global_settings) -> bool:
+        if global_settings[Tags.DIM_VOLUME_Z_MM] < (self.probe_height_mm + 1):
```

Comment:
Here we would add 1mm on both sides independently of the spacing. Better change this to one voxel